modules/dnn/src/tensorflow/tf_importer.cpp

```diff
@@ -1469,11 +1469,29 @@ void TFImporter::populateNet(Net dstNet)
 
             layerParams.set("height", outSize.at<int>(0, 0));
             layerParams.set("width", outSize.at<int>(0, 1));
+            layerParams.set("interpolation", 0);
```

Comment:
Enumerations `INTER_NEAREST` and `INTER_LINEAR` are also integers. You can just set them as arguments, `layerParams.set("interpolation", INTER_NEAREST);` and use `interpolation` like `resize(getPlane(inp, n, ch), getPlane(out, n, ch), Size(outWidth, outHeight), 0, 0, interpolation);`